admin/app/controllers/workarea/admin/content_blocks_controller.rb

```diff
@@ -52,6 +52,22 @@ def move
       head :ok
     end
 
+    def copy
+      new_block = @block.clone
+      new_block.content = @block.content
+
+      if new_block.save
+        flash[:success] = t('workarea.admin.content_blocks.flash_messages.copied')
+      else
+        flash[:error] = new_block.errors.to_a.to_sentence
```

Comment:
think you want `errors.full_messages.to_sentence` ?

Reply:
💔

Reply:
agreed, it will render more descriptive errors.